x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/inference/ltr/QueryFeatureExtractorTests.java

```diff
@@ -125,4 +128,29 @@ public void testQueryExtractor() throws IOException {
         dir.close();
     }
 
+    public void testEmptyDisiPriorityQueue() throws IOException {
```

Comment:
@kderusso @benwtrent  Added this test that is failing on main (as expected) but not on the branch.